### PR TITLE
optimize(tests/logs): add newline to format logs

### PR DESCRIPTION
**Pull request by 1046102779**
Signed-off-by: 1046102779 <seachen@tencent.com>

closed PR: https://github.com/dapr/dapr/pull/5338

### CODE REVIEW

**Comment by 1046102779** [comment]
cc @daixiang0 @artursouza

**Comment by daixiang0** [comment]
@1046102779 LGTM overall, please fix conflicts.

**Comment by 1046102779** [comment]
> @1046102779 LGTM overall, please fix conflicts.

Done.

**Review by mukundansundar — DISMISSED** [review]
lgtm.

File: tests/apps/actorfeatures/app.go [hunk 1]
```diff
@@ -202,6 +202,7 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 
 func logsHandler(w http.ResponseWriter, r *http.Request) {
 	log.Printf("Processing dapr %s request for %s", r.Method, r.URL.RequestURI())
+	log.Printf("Processing dapr %s request for %s\n", r.Method, r.URL.RequestURI())
```
[the remark this answers is not in the document]
Comment (1046102779) [reply tests/apps/actorfeatures/app.go:204]: Thanks. Very detailed @mukundansundar .

**Review by mukundansundar — DISMISSED** [review]
lgtm

**Review by ItalyPaleAle — APPROVED** [review]

**Comment by ItalyPaleAle** [comment]
@1046102779 thanks for this contribution, however I don't believe this is needed.

Unlike `fmt`, `log` already adds newlines. Example:

image

**Pull request closed without merging**